bucket.go

```diff
@@ -22,7 +22,7 @@ const InfiniteRetention = 0
 type Bucket struct {
 	ID                  ID            `json:"id,omitempty"`
 	OrganizationID      ID            `json:"orgID,omitempty"`
-	Organization        string        `json:"organization,omitempty"`
+	Org                 string        `json:"org,omitempty"`
```

Comment:
Do we even need this field on the Bucket struct? We had initially added it because we wanted to present things nicely going out the door for the API, but we've since developed patterns for doing presentation in the http layer. Might be worthwhile to simply remove this field all together.

Reply:
Yeah, I agree. We should remove it, in that case.